lib/web_ui/lib/src/engine/pointer_binding/event_position_helper.dart

```diff
@@ -41,12 +41,12 @@ ui.Offset computeEventOffsetToTarget(
   if (isInput) {
     final EditableTextGeometry? inputGeometry = textEditing.strategy.geometry;
     if (inputGeometry != null) {
-      return _computeOffsetForInputs(event, inputGeometry);
+      return _computeOffsetForInputs(event, eventTarget, inputGeometry);
```

Comment:
From the docs:

> This needs an `eventTarget`, because the `event.target` (which is what this would really need to use) gets lost when the `event` comes from a "coalesced" event (see: flutter/flutter#155987).

Reply:
I'll update the docs. The problem I'm seeing in Chrome is actually quite interesting:

The main event has the correct target (input). But the coalesced events have a dummy div as their targets. When I set a breakpoint in Chrome and try to inspect that div, Chrome freezes!

I filed a Chrome bug for it: https://g-issues.chromium.org/issues/382473107